spec/fixtures/application/1.0/application.json

```diff
@@ -98,9 +98,7 @@
       "income_above_threshold": "no",
       "has_frozen_income_or_assets": "no",
       "client_owns_property": "no",
-      "has_savings": "no",
-      "manage_without_income": "other",
-      "manage_other_details": "Another way they manage",
+      "has_savings": "yes",
```

Comment:
So it is more consistent with the business logic (if all are no, no outgoings or capital qs should be asked)